.eslintrc

```diff
@@ -1,79 +1,155 @@
 {
   'parser': 'babel-eslint',
+  'extends': 'airbnb',
   'plugins': [
-    'react'
+     'redux-saga',
+     'react',
+     'jsx-a11y',
+     'markdown'
   ],
   'env': {
+    'mocha': true,
     'browser': true,
-     'node': true,
-     'es6': true
+    'node': true,
+    'es6': true
   },
   'globals': {
         'React': true
     },
   'ecmaFeatures': {
       'jsx': true
   },
+  'parserOptions': {
+    'ecmaVersion': 6,
+    'sourceType': 'module',
+    'ecmaFeatures': {
+      'jsx': true
+    }
+  },
   'rules': {
-    # Error rules
-    'no-dupe-args': 2,
-    'no-dupe-keys': 2,
-    'no-empty': 2,
-    'no-extra-parens': 1,
-    'no-extra-semi': 1,
-    'no-irregular-whitespace': 1,
-    # Best Practices rules
+    'array-bracket-spacing': [ 1, 'always' ],
     'array-callback-return': 1,
+    'arrow-spacing': [ 1, {
+      'before': true,
+      'after': true
+    }],
+    'arrow-body-style': [
+      2,
+      'as-needed'
+    ],
     'block-scoped-var': 1,
+    'block-spacing': 1,
```

Comment:
IME, warnings will be ignored and will just be noise. can we make them errors / fix violations or just remove the rule?